src/oc_erchef/raml-docs/README.md

```diff
@@ -12,7 +12,20 @@ plans to make these docs official or comprehensive.
 
 ## Building HTML Docs From RAML Source
 
-1) swallow your pride and get node.js
-2) `npm install -g raml2html`
-3) `raml2html SOURCE_FILE.yml > OUTFILE.html`
+### Prerequisites
 
+`raml2html` is a node.js joint, so swallow your pride and get node.js
```

Comment:
It was there before, and not mine to remove :P, you'll have to take it up with @danielsdeleo